index.js

```diff
@@ -75,13 +76,40 @@ const Home = () => {
   const [fetchError, setFetchError] = react.useState(false);
   const [versions, setVersions] = react.useState([]);
 
+  function reducer(state, action) {
+    switch (action.type) {
+      case 'toggle':
+        return { isSearching: !state.isSearching };
+      case 'open':
+        return { isSearching: true };
+      case 'close':
+        return { isSearching: false };
+      default:
+        throw new Error();
+    }
+  }
+
+  const [state, dispatch] = react.useReducer(reducer, { isSearching: false });
+
+  react.useEffect(() => {
+    const check = e => {
+      if (e.key === 'p' && e.metaKey) {
```

Comment:
think this should be `... && (e.metaKey || e.ctrlKey)` for Windows/Linux users. Sorry if I forgot to put it in! Might not have pushed it...

Reply:
Fixed